common/constants.js

```diff
@@ -52,6 +52,7 @@ module.exports = Object.freeze({
     UPDATE_EVENT: 'update_instance'
   },
   EVENTMESH_POLLER_DELAY: 200,
+  CLUSTER_SCHEDULER_DELAY: 50,
```

Comment:
Isn't it too low? let us keep 200 ms

Reply:
Delay will be too much if we keep it 200 I feel. `100`?

Reply:
okay